mud-contracts/world-v2/src/namespaces/evefrontier/systems/deployable/DeployableSystem.sol

```diff
@@ -197,13 +210,22 @@ contract DeployableSystem is SmartObjectFramework {
    */
   function anchor(
     uint256 smartObjectId,
+    address owner,
     LocationData memory locationData
   ) public onlyActive context access(smartObjectId) scope(smartObjectId) {
     State previousState = DeployableState.getCurrentState(smartObjectId);
     if (previousState != State.UNANCHORED) {
       revert Deployable_IncorrectState(smartObjectId, previousState);
     }
     _setDeployableState(smartObjectId, previousState, State.ANCHORED);
+    // ascribe ownership tracking of the deployable smart object
+    address currentOwner = ownershipSystem.owner(smartObjectId);
+    if (currentOwner != address(0) && currentOwner != owner) {
+      ownershipSystem.annulFromAccount(smartObjectId, currentOwner);
+      ownershipSystem.ascribeToAccount(smartObjectId, owner);
+    } else if (currentOwner == address(0)) {
```

Comment:
might be best if the cases are swapped as the most used case is `currentOwner == address(0)` so it does not have to go to else in most cases